src/Auth.js

```diff
@@ -66,6 +68,47 @@ function nobody(config) {
   return new Auth({ config, isMaster: false });
 }
 
+const throttle = {};
+const renewSessionIfNeeded = async ({ config, session, sessionToken }) => {
+  if (!config?.extendSessionOnUse) {
+    return;
+  }
+  clearTimeout(throttle[sessionToken]);
+  throttle[sessionToken] = setTimeout(async () => {
```

Comment:
throttle seems to be a simple object never cleared, and it's not an LRU, memory is exposed to DDOS issues (since it seems that every token will be stored in memory), and the memory will grow infinitely

Reply:
I checked also clearTimeout function do not unset in the record on the throttle object. So the timeout instance is cancelled but still there in memory

Reply:
I’ll open a new issue and fix

Reply:
I checked also clearTimeout function do not unset in the record on the throttle object. So the timeout instance is cancelled but still there in memory